overlord/snapstate/snapmgr.go

```diff
@@ -330,14 +330,14 @@ func updateInfo(st *state.State, snapst *SnapState, channel string, userID int,
 	return res[0], nil
 }
 
-func snapInfo(st *state.State, name, channel string, userID int, flags Flags) (*snap.Info, error) {
+func snapInfo(st *state.State, name, channel string, rev snap.Revision, userID int, flags Flags) (*snap.Info, error) {
```

Comment:
rev, revno, revision.. can we please stick to "revision" when there are no conflicts?